openapi/components/schemas/GatewayAccountConfig/Adyen.yaml

```
@@ -73,6 +73,10 @@ allOf:
             format: double
             description: Total tax amount as a decimal to send with level 2 enhanced scheme data.
             example: 0.06
+          enableMoto:
+            type: boolean
+            description: Specifies whether to use Mail Order/Telephone Order (MOTO).
```

```suggestion
            description: Specifies whether to use Mail Order Telephone Order (MOTO).
```

Rollback the change to be consistent with Bluesnap